tidy3d/components/medium.py

```diff
@@ -1241,6 +1303,13 @@ def eps_diagonal(self, frequency: float) -> Tuple[complex, complex, complex]:
         eps_zz = self.zz.eps_model(frequency)
         return (eps_xx, eps_yy, eps_zz)
 
+    def eps_comp(self, row: Axis, col: Axis, frequency: float) -> complex:
+        """Single component the complex-valued permittivity tensor as a function of frequency."""
+
+        if row == col:
+            return [self.xx, self.yy, self.zz][row].eps_model(frequency)
```

Comment:
could consider using `self.components` (dict of `"ii"` to `self.ii`) like so
```py
if row != col:
    return 0j
cmp = "xyz"[row]
field_name = cmp + cmp
return self.components[field_name].eps_model(frequency)
```
but what you have is also ok. Just thinking if AnisoptricMedium is changed later it's slightly nicer to use the .components property for generality

Reply:
stole this